### DISCUSSION TITLE
How to add border to abbreviation help info

### DISCUSSION THREAD

**Discussion by ATMxsp01** [Julian/lean.nvim#365] [General]
I'd like to add a border to the abbreviation help information. Is there a way to do so?

**Comment by Julian** [comment]
There's no user-facing API for customizing this (yet), but I added one to match us adding one elsewhere in 775c1e0. Feel free to update and let me know if this does what you want. And thanks for flagging.

**Reply by ATMxsp01** [reply]
It works very well, thank you!
